kinto-remote-settings/src/kinto_remote_settings/signer/listeners.py

```diff
@@ -630,7 +630,7 @@ def cleanup_preview_destination(event, resources):
             continue
 
         for k in ("preview", "destination"):
-            if k not in resource:  # pragma: nocover
+            if k not in resource:  # pragma: no cover
```

Comment:
I noticed that we used `nocover` in a few other places. If this is the correct way to mark a line as no cover, can we do a global find + replace?

Reply:
Also, should we not cover this line, or should we add a test case that covers this branch?

Reply:
It is relatively acceptable to ignore this branch. 

To cover it, we should add a test that cleanups a resource which hasn't any preview (`source -> destination`)